src/app/users/users-achievements/users-achievements.component.html

```diff
@@ -41,5 +41,6 @@ <h4 *ngIf="achievements[infoType]">{{infoType}}</h4>
         </mat-list>
       </div>
     </div>
+    <div *ngIf="!achievements">You haven't entered any achievements yet. Click Edit Achievements to add your achievements!</div>
```

Comment:
On a slow connection this message will show while the request to get the achievements doc completes.  We should not show the message until after the request completes.